RG3205W_rev5_howto.en.md

```diff
@@ -0,0 +1,202 @@
+
+If you instead prefer to simply dim the display down, the *Screen Timeout* app (`de.lhoer0.screentimeout`) is what you're looking for.
+You could achieve the same through the settings of the original Movistar apps but, interestingly, the same menu is disabled from the Android configuration.
+Crucially, when the display goes idle due to inactivity, the Android OS doesn't send the broadcast intent to communicate the screen has shut down,
+much like if that bit of the OS has been stripped out. This means that, sadly, you can't have either the screen lock, or the screensaver start,
+right when the display dims: the OS feature has been stripped. This leads to the below automation headache.
+
+### Putting the screen to sleep
+I ended up installing *Automate* (`com.llamalab.automate`), that now manages the screensave/screen lock, and a few other bits (see below).
+
+Since we can't tell when the display dims or, in general, when the device is idle, I've automated a workaround that, well, works, albeit it doesn't make me proud.
+
+The automation starts, waits for 3 minutes (here you can pick the average amount of time you think you'll be continuosuly interacting with the screen),
+and then just launches the screensaver (you want to pick the `Somnambulator` activity) -- alternatively you could lock the screen.
+Then, the automation waits for the `DREAMING_STOPPED` broadcast intent (if you decided to lock the screen instead, pick the proper broadcast event on wake up),
+which triggers once someone touches the screen, waking the device up. At which points, the automation loops on the 3 minutes wait.
+Not clean as I'd like, but it works *just good enough*.
+
+You'll want to configure Automate to resume your automations at boot, so they keep running if the power goes out.
+But, yes, you read well: Automate will *resume* automations, not *restart* them at boot. What does this means?
+Suppose your device was idle, with the screensaver active, and the power goes down. When the power comes back up, your device reboots... and the
+screensaver never kicks in. This is because with the screensaver active, the automation was waiting for the `DREAMING_STOPPED` intent. After reboot,
+the device has no screensaver active, so unless you run and then exit from it manually, the `DREAMING_STOPPED` intent will never be sent, deadlocking your resumed automation.
+
+Current dirty solution: create another automation that waits for the `BOOT_COMPLETED` intent, then launches the Screen Saver, and loops waiting again for the boot intent.
+(Another solution could be: create an automation that, on boot, kills and re-start all other automations -- requires more nodes that eat against the Automate free tier.)
+Yes, you see why I can't be proud. But, again, it works, within the limitations of the OS running on the Movistar Home.
+
+If you find any way of programmatically detecting when the device is idle, please let the community know!
+
```

Comment:
Is it possible to export & import the complete workflows from Automation as file? So others can replicate them easier.

Reply:
I'm working on it. I could dump individual flows, but need to check they restore properly. I also need to sanitize them (e.g. set a dummy location for the sunrise/sunset calculation).

Anyway, I do intend to share them. We could update this page once they're ready?

Reply:
Alright, just added the three flows I'm using (see 3rd commit at the bottom of this PR). If you're happy to merge now, I'll then prepare a further PR to update the text to link to the files and explain how to import and configure them in Automate, to make it more understandable.